android/src/main/java/com/applovin/reactnative/AppLovinMAXModuleImpl.java

```diff
@@ -1581,6 +1583,8 @@ private void setAdViewExtraParameters(final String adUnitId, final MaxAdFormat a
             }
             else if ( "adaptive_banner".equalsIgnoreCase( key ) )
             {
+                e( "Setting adaptive banners via extra parameters is discouraged. Please use the `BannerAd.createAd(adUnitId: string, position: AdViewPosition, xOffset: number, yOffset: number, isAdaptive: boolean)` API for proper adaptive banner configuration." );
```

Comment:
I actually think we should just be more aggressive here and note that this method is deprecated and removed in a future plugin version.

Then, we need to update [docs](https://developers.applovin.com/en/max/react-native/ad-formats/banner-and-mrec-ads#adaptive-banners) to indicate as such as well.

Reply:
Then when the time arises where we remove this method of integration from the test app, we can remove `disabledAdaptiveBannerAdUnitIds` as well which is our stance of not being able to mutate configurations mid-session.

Reply:
Thanks, I updated the error message and will file a doc tikect.